src/collections/infra/repositories/transformers/collectionTransformers.ts

```diff
@@ -0,0 +1,18 @@
+import { Collection } from '../../../domain/models/Collection'
+import { AxiosResponse } from 'axios'
+import { CollectionPayload } from './CollectionPayload'
+
+export const transformCollectionIdResponseToPayload = (response: AxiosResponse): Collection => {
```

Comment:
```suggestion
export const transformCollectionResponseToCollection = (response: AxiosResponse): Collection => {
```

I think you mean transforming the Collection response or payload to the Collection model

Reply:
Yep, you're right. Started with just ID and neglected to change this. Should be OK now